docs/apps/bs.md

```diff
@@ -0,0 +1,46 @@
+
+# Illumina BaseSpace
+
+## Description
+
+llumina BaseSpace command line client,`bs`, can be used to retrievie data from the Illumina BaseSpace environment to Puhti.
+
+## Available
+
+*    bs version 1.0.0 is available in Puhti
+
+## Usage
+
+The Illumina BaseSpace command line client starts with command:
+```text
+bs
+```
```

Comment:
The `bs` command is not in path. Does the user need to load/set/install something for these instructions to work?

Reply:
korjattu